### ISSUE TITLE
Allow opt-in to restore NVM backups without checking for compatibility with the target controller

### ISSUE THREAD

**Issue by umtauscher** [zwave-js/zwave-js-ui#3336]
### Checklist

- [ ]  I am not using Home Assistant. Or: a developer has told me to come here.
- [ ] I have checked the troubleshooting section and my problem is not described there.
- [ ] I have read the changelog and my problem is not mentioned there.

### Deploy method

Docker

### Z-Wave JS UI version

2.0.1

### ZwaveJS version

0.1.93

### Describe the bug

I made an NVM Backup from my currentlich running Aeotec Gen5 and tried to restore that backup to aother Stick of the same make and kind.
The restore stated with "converting ..." (for whatever reson) and after reaching 100% it stated theat it couln't find a matching converter.
The restore failed.

### To Reproduce

Just as I described above

### Expected behavior

to get a copy of my setup on a second stick

### Additional context

Homeassistant 2023.3.9  on OS10,5 using supervisor 2023.09.2

**Comment by robertsLando** [comment]
@umtauscher If you are using docker and you don't want to wait for next release, you should be able to test the fix on master by using `master` tag in one hour from now

**Comment by umtauscher** [comment]
Tanks, unfortunaltely I will stay with my existing instalation. 
I 'll wait, I have just restored to my backup stick with the old Aeotec tool. So I have time.
Thanks for the proposal anyways.
Cheers

**Comment by umtauscher** [comment]
Btw, just a quick question: If I risc installing the 1.02 firmware to the stick (Aeotc themself warn about bricking it in the process) would I then be able to use the normal restore function?

**Comment by AlCalzone** [comment]
Yes, that one is supported - and it can also be migrated to newer sticks, unlike the 1.1 NVM.

**Comment by umtauscher** [comment]
Sorry, but dues to the zwave updates NVM-backup is now unusable for this configuration.
Tonight was the second time that NVM Backup froze the stick.
Should I open another thread for it youwould you like to continue here.

I was using NVM backup für at least a year without problems but something seems to be messes up.
Sorry

**Comment by AlCalzone** [comment]
You can send me a log via discord, so I can look if that's a bug or usage error

**Comment by umtauscher** [comment]
All logs were disabled this night, sorry.
What usage error could cause this?
I am running homeasistant on a proxmox server. To get zwave back, I had to shut down the server completely, because the port the stick was using was nonexistant any more. A software restart wasn't enough, neither was unplugging and replugging the stick. I seemd as if the power on the usb port had been switched off.

I have deactivated nvm backup for now, which was running before every sunday at 3 am.
